### PR TITLE
new log files

### PR DESCRIPTION
**Pull request by sandeep9gss**
Create

### CODE REVIEW

File: data.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+THis is git class
+THis is git class
+THis is git class
```
Comment (AmarnathAitha) [inline data.log:3]: I do not approve

File: test.log [hunk 2]
```diff
@@ -0,0 +1 @@
+this is log file
```
Comment (AmarnathAitha) [inline test.log:1]: test

File: data.log (hunk 1 above)
Comment (sandeep9gss) [reply data.log:3]: okay i am going to remove duplicate code

File: test.log (hunk 2 above)
Comment (sandeep9gss) [reply test.log:1]: ok

**Review by AmarnathAitha — APPROVED** [review]
Approve

**Pull request merged**